ext/tokenizer/tests/PhpToken_constructor.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+PhpToken constructor
+--FILE--
```

Comment:
Nit: add a SKIPIF section for these tests for builds with ./configure --disable-tokenizer

```suggestion
--SKIPIF--
<?php if (!extension_loaded("tokenizer")) print "skip tokenizer extension not enabled"; ?>
--FILE--
```